content/en/docs/ops/ambient/usage/ztunnel/_index.md

```diff
@@ -0,0 +1,498 @@
+istio-cni-node   3         3         3       3            3           kubernetes.io/os=linux   6h34m
+ztunnel          3         3         3       3            3           <none>                   6h35m
+{{< /text >}}
+
+{{< text bash >}}
+$ kubectl get istiooperator/installed-state -n istio-system -o yaml
+{{< /text >}}
+{{< text syntax=plain snip_id=none >}}
+--snip--
+profile: ambient
+  tag: 1.18.0
+  values:
+    base:
+      enableCRDTemplates: false
+      validationURL: ""
+    cni:
+      ambient:
+        enabled: true
+    ztunnel:
+      enabled: true
+  hub: docker.io/istio
+  meshConfig:
+    defaultConfig:
+      proxyMetadata:
+        ISTIO_META_ENABLE_HBONE: "true"
+--snip--
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+The output of `istioctl verify-install` should indicate all items installed successfully including some ztunnel components as indicated in the examples.
+
+If `ambient` is installed correctly, you should see 1 instance of ztunnel proxy pod per node in RUNNING state in the cluster (including control plane nodes and worker nodes). You should also see 1 instance of the `istio-cni` pods per node and a single instance of the `istiod` controller pod per cluster, all in RUNNING state. 
+
+Confirm from the `istioOperator` output that profile is normally set to `ambient` (unless a custom profile is being used), ztunnel is set to enabled,  cni is enabled for `ambient`. Notice also proxyMetaData field has ISTIO_META_ENABLE_HBONE set to true. If using a custom installation profile, these fields must be set as described to enable `ambient` mode within a custom profile. It is recommended to start with the built-in `ambient` profile before trying any custom variations.
+
+[//]: # ( ## NOTE Before we go further, would be good to have a section on roles, RBAC, operational model )
+
+## Functional Overview
+
+The figure shows an architecture summary of the Ztunnel proxy function.
+
+{{< image width="100%"
+    link="ztunnel-architecture.png"
+    caption="Ztunnel architecture"
+    >}}
+
+For additional architecture details, refer to the Ztunnel architecture guide. For now we mainly note that each instance of the ztunnel proxy uses the Envoy xDS apis to receive certificates, discovery and configuration information from the istio control plane (`istiod`) on behalf of all pods and endpoints associated with it. In particular the ztunnel proxy obtains M-TLS certificates for all Service accounts of all pods that are associated with it. Since a single ztunnel proxy performs both the data plane and the control plane operations across multiple service accounts, it is a multi-tenant component of the mesh infrastructure in contrast with Istio side car proxies that handle control plane and data plane operations on a per application endpoint or pod basis.
+
+<< To add some content on the xDS api objects of relevance to Ambient/ ztunnel >>
+
+## Deploying an Application
```

Comment:
```suggestion
## Deploying an Application
```

I would suggest breaking this up into 2 separate back-to-back sections for better clarity and impact:

`## Deploying Applications`

`## Adding Deployed Applications To The Ambient Mesh`

Reply:
Okay, making this change.